pilot/pkg/serviceregistry/kube/controller/endpointslice.go

```diff
@@ -114,7 +114,7 @@ func (esc *endpointSliceController) onEventInternal(_, ep *v1.EndpointSlice, eve
 	if svc != nil && !serviceNeedsPush(svc) {
 		return
 	}
-	log.Infof("triggering EDS push for %s %s in namespace %s", name, event, namespace)
+	log.Debugf("triggering EDS push for %s %s in namespace %s", name, event, namespace)
```

Comment:
The log need to adjust. move  after L119

log.Debugf("triggering EDS push for %s  in namespace %s", hostnames, namespace)